packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts

```diff
@@ -259,39 +284,38 @@ export class PushNotificationsService {
 
             return {
               copayerId: p.copayerId,
-              language: p.language,
+              language: p.language || this.defaultLanguage,
               unit: unit || p.unit || this.defaultUnit
             };
           })
         );
 
         const copayers = _.keyBy(recipientPreferences, 'copayerId');
 
-        const recipientsList = _.compact(
-          _.map(wallet.copayers, copayer => {
-            if (
-              (copayer.id == notification.creatorId && notificationType.notifyCreatorOnly) ||
-              (copayer.id != notification.creatorId && !notificationType.notifyCreatorOnly)
-            ) {
-              const p = copayers[copayer.id] || {
-                language: this.defaultLanguage,
-                unit: this.defaultUnit
-              };
-              return {
-                copayerId: copayer.id,
-                language: p.language || this.defaultLanguage,
-                unit: unit || p.unit || this.defaultUnit
-              };
-            }
-          })
-        );
-
+        const recipientsList = wallet
```

Comment:
intead of this `wallet` check and this https://github.com/bitpay/bitcore/pull/3027/files#diff-516ac13e45f6e22ca8d66a9b72e8d20851abddac3e12637d389862a8b95295e5R270


maybe add:

`if (!wallet) return cb(null, []); `

at: https://github.com/bitpay/bitcore/pull/3027/files#diff-516ac13e45f6e22ca8d66a9b72e8d20851abddac3e12637d389862a8b95295e5R268